File: Day-11 ( Simple Calculator Using Switch Case )/CPP/Solution.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    char op;

    cout << "Enter first number: ";
    cin >> a;

    cout << "Enter second number: ";
    cin >> b;

    cout << "Enter operator (+, -, *, /, %): ";
    cin >> op;

    switch (op) {
        case '+':
            cout << a << " + " << b << " = " << a + b << endl;
            break;

        case '-':
            cout << a << " - " << b << " = " << a - b << endl;
            break;

        case '*':
            cout << a << " * " << b << " = " << a * b << endl;
            break;

        case '/':
            if (b == 0) {
                cout << "Cannot divide by zero" << endl;
            }
            else {
                cout << a << " / " << b << " = " << a / b << endl;
            }
            break;
            
        case '%':
            if (b == 0) {
                cout << "Cannot divide by zero" << endl;
            }
            else {
                cout << a << " % " << b << " = " << a % b << endl;
            }
            break;

        default:
            cout << "Invalid operator" << endl;
    }

    return 0;
}

File: Day-11 ( Simple Calculator Using Switch Case )/Python/Solution.py
a = int(input("Enter first number: "))
b = int(input("Enter second number: "))
op = input("Enter operator (+, -, *, /, %): ")

match op:
    case "+":
        print(f"{a} + {b} = {a + b}")

    case "-":
        print(f"{a} - {b} = {a - b}")

    case "*":
        print(f"{a} * {b} = {a * b}")

    case "/":
        if b == 0:
            print("Cannot divide by zero")
        else:
            print(f"{a} / {b} = {a / b}")

    case "%":
        if b == 0:
            print("Cannot divide by zero")
        else:
            print(f"{a} % {b} = {a % b}")

    case _:
        print("Invalid operator")